src/scene/board/property-controller.ts

```diff
@@ -0,0 +1,103 @@
+import BoardSceneController from "@/scene/board-scene-controller.ts";
+
+const MAX_PROPERTY_LEVEL = 3;
+
+const DEFAULT_PROPERTY_PRICE = 100;
+const PROPERTY_PRICE_MULTIPLIER = 1.5;
+
+const DEFAULT_PROPERTY_RENT = 50;
+const PROPERTY_RENT_MULTIPLIER = 1.5;
+
+/**
+ * HOST only class
+ * PropertyController is responsible for managing property related actions
+ */
+export default class PropertyHelper {
```

Comment:
something like "PropertyManager" would be a better name.
"Helper" indicates a util class, which this is not